app/src/main/java/com/android/sample/ui/authentication/Welcome.kt

```diff
@@ -112,19 +93,11 @@ fun SignUpButton(navigationActions: NavigationActions) {
             modifier = Modifier.fillMaxSize(),
             colors =
                 ButtonDefaults.buttonColors(
-                    containerColor = Color.White, contentColor = PrimaryPurple),
+                    containerColor = PrimaryWhite, contentColor = PrimaryPurple),
```

Comment:
Consider using the colors from the MaterialTheme.colorScheme to maintain consistency.

```suggestion
                    containerColor = MaterialTheme.colorScheme.background, contentColor = MaterialTheme.colorScheme.primary),
```